src/components/Card.js

```diff
@@ -1,9 +1,23 @@
 export default class Card {
-  constructor( data, cardSelector, handleImageClick) {
+  constructor(
+    data,
+    cardSelector,
+    handleImageClick,
+    removeLike,
+    isLiked,
+    _id,
+    handleDeleteClick,
+    likeCard
+  ) {
     this._name = data.name;
     this._link = data.link;
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._removeLike = removeLike;
+    this._isLiked = isLiked;
+    this.id = _id;
+    this._handleDeleteClick = handleDeleteClick;
+    this._likeCard = likeCard;
   }
 
   getView() {
```

Comment:
Needs correction: don't forget to call ` renderLikes` inside `getView` method, so you will show the initial likes state